### ISSUE TITLE
Build error with relative paths

### ISSUE THREAD

**Issue by ZeAntwan** [midouest/vscode-playdate-debug#35]
When executing the "Debug" action, building the game and running it in the simulator, if any files contains an import with a relative link to a parent folder, this will lead to the build failing with this error:

```
Compiling...
> "D:\PlaydateSDK\bin\pdc" -sdkpath "D:\PlaydateSDK" "d:\PlaydateSDK\Examples\ExtensionError\source" "d:\PlaydateSDK\Examples\ExtensionError\source.pdx"
Command failed: "D:\PlaydateSDK\bin\pdc" -sdkpath "D:\PlaydateSDK" "d:\PlaydateSDK\Examples\ExtensionError\source" "d:\PlaydateSDK\Examples\ExtensionError\source.pdx"
error: d:\PlaydateSDK\Examples\ExtensionError\source/main.lua:5: No such file: ../test
```

My test setup is simple:
```
test.lua
source
|__main.lua
|__game.lua
```
with a simple print in both "game" and "test" and the "main.lua" files contains this
```lua
import 'game'
import '../test'
```

Building the game manually with pdc works as expected

I am using Window 10, and this has been my only issue so far with it

**Comment by ZeAntwan** [comment]
I did a bit of investigation, and it seems to be an issue because the extension triggers a command with "\" in the path, where I'm using "/" in code.
You can see in this screenshot that I managed to run the same command by just changing the slashes to get it to work.
Could that be a PDC problem rather than an extension one?
![image](https://github.com/midouest/vscode-playdate-debug/assets/11023019/2676aaa9-0f5a-47dd-8708-4f42014435f2)

**Comment by ZeAntwan** [comment]
Actually, the original command seems to work only when my command prompt is already navigated inside the projects folder
![image](https://github.com/midouest/vscode-playdate-debug/assets/11023019/0ea4fbb1-1f7c-4a74-b0d0-f606dea4fa84)

**Comment by midouest** [comment]
This should be fixed in version 1.16.0 of the extension, available shortly! Thanks for the report!
